lib/portlayer/exec/vchinfo.go

```diff
@@ -19,12 +19,14 @@ import (
 
 	"github.com/vmware/govmomi/vim25/mo"
 	"github.com/vmware/govmomi/vim25/types"
+	log "github.com/Sirupsen/logrus"
+	"fmt"
```

Comment:
goimports -w -local github.com/vmware $(find . -mindepth 1 -maxdepth 1 -type d -not -name vendor)